aggregate_spatial_window.json

```
@@ -0,0 +1,95 @@
+{
+  "id": "aggregate_spatial_window",
+  "summary": "Zonal statistics for rectangular windows",
+  "description": "Aggregates statistics over the spatial dimensions. This process passes a list of values to the reducer.\n\nThe results are computed dividing the input X/Y grid into x_size/y_size non-overlapping windows and applying the reducer process to each window.\nIf X and Y are not multiple of x_size/y_size NaNs padding before the computation is applied.",
```

Trying to align a bit more with other processes; can probably still be improved:
```suggestion
  "description": "Aggregates statistics over the horizontal spatial dimensions (axes `x` and `y`) of the data cube.\n\nThe pixel grid for the axes `x` and `y` is divided into non-overlapping windows with the size specified in the parameters `x_size` and `y_size`. If the number of values for the axes `x` and `y` is not a multiple of the corresponding `x_size` or `y_size` parameter, the behavior specified in `boundary` parameter is applied.\nFor each of these windows, the reducer process computes the result.",
```